src/Fixer/ClassNotation/FinalInternalClassFixer.php

```diff
@@ -156,7 +142,7 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
         $annotationsNormalizer = static function (Options $options, array $value): array {
             $newValue = [];
             foreach ($value as $key) {
-                if ('@' === $key[0]) {
+                if (str_starts_with($key, '@')) {
```

Comment:
`str_starts_with` is available in PHP8+ but Fixer is for 7.4+, so it can't be used here.

Reply:
There is, we require `symfony/polyfill-php80`.